docs/build-on-linea/linea-version/index.mdx

```diff
@@ -55,6 +55,16 @@ To decrease L1 costs, we implemented the following features:
 - Switched L2 to L1 messaging anchoring from individual hashes to Merkle trees:
   - Due to the size of the Merkle trees (32 messages) requiring only one Merkle root to be anchored, there is a 32x calldata size reduction for message hashes
 
+### Smart contract changes
+
+To support these new features, we implemented changes to our smart contracts. The audited commit for this update can be found [here] (https://github.com/Consensys/linea-contracts-audit/commit/99039ebc8d6cb3009cf46286d5de4c484e03bc81). 
+
+Two independant audits have been perfomed for this update. Reports can be found here: 
```

Comment:
need to change "independant" to "independent" and "perfomed" to "performed"

Reply:
Thanks, updated 👍